lib/filter-schema.js

```diff
@@ -0,0 +1,140 @@
+'use strict'
+
+const {
+  wrapSchema,
+  PruneSchema,
+  FilterTypes,
+  TransformObjectFields
+} = require('@graphql-tools/wrap')
```

Comment:
graphql-tools is not a depdency. Could we avoid it? Or just add it as a dependency?

Reply:
I have added it to the package.json

Reply:
How feasible would it be to not include this dependency?

Reply:
I have checked it and it requires a lot of GraphQL knowledge about the spec and the `graphqljs` implementation to manage all the use cases.

I'm monitoring this issue: https://github.com/graphql/graphql-js/issues/113
Or it could be best to include this feature in the official graphql implementation.

At the moment the issue is 5 years old but still active (people want it)